nnpdf_data/nnpdf_data/filter_utils/eic_utils.py

```diff
@@ -0,0 +1,206 @@
+from pathlib import Path
+from typing import Optional, Union
+
+import numpy as np
+import pandas as pd
+import yaml
+
+
+def read_cvs(path: Path) -> np.ndarray:
```

Comment:
```suggestion
def read_central_values(path: Path) -> np.ndarray:
```

I misread this as `read_csv`